### ISSUE TITLE
Discuss: is this repo being used?

### ISSUE THREAD

**Issue by afred** [WGBH/stock-sales-admin#11]
@mccalluc?

**Comment by mccalluc** [comment]
No: original idea was: 'RoR admin site for WGBH Stock Sales'. You can delete this, or I can.

**Comment by afred** [comment]
:+1: go ahead. If there is anything of value to you in the comments, tickets, etc. be sure to grab it first.
